packages/xod-client/src/utils/normalizeColor.js

```diff
@@ -0,0 +1,11 @@
+import * as R from 'ramda';
+import { DEFAULT_VALUE_OF_TYPE, CUSTOM_TYPE } from 'xod-project';
+
+export default R.ifElse(
```

Comment:
Shouldn't it also automatically insert the leading `#` if it's missing? (`FFD406 ` -> `#FFD406 ` or even `FFF` -> `#FFFFFF`)?